src/sota_tools/deploy.cc

```diff
@@ -77,18 +81,18 @@ bool OfflineSignRepo(const ServerCredentials &push_credentials, const std::strin
     boost::filesystem::remove_all(local_repo);
   }
 
-  std::string init_cmd("garage-sign init --repo aktualizr --credentials ");
+  std::string init_cmd("./garage-sign/bin/garage-sign init --repo aktualizr --credentials ");
```

Comment:
I would switch back to `init_cmd("garage-sign init --repo aktualizr --credentials ");`, this would make problems in the actual released code.